src/OrbitQt/MultipleOfSpinBoxTest.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include "MultipleOfSpinBox.h"
+
+namespace orbit_qt {
+TEST(MultipleOfSpinBox, Validate) {
+  MultipleOfSpinBox spin_box{nullptr};
+  spin_box.setSingleStep(8);
+  spin_box.setMinimum(16);
+  spin_box.setMaximum(512);
+
+  int pos = 0;
+
+  QString sixteen = QString::fromStdString("16");
```

Comment:
Nit: Don't use `fromStdString` if your input type is not a std::string. Use `QStringLiteral()` instead.